contribs/gmf/less/mobile.less

```diff
@@ -227,15 +227,24 @@ gmf-map {
 }
 
 .ol-control button,
-.ol-touch .ol-control button {
-  font-size: 1em;
+.ol-touch .ol-control button,
+.fa:after,
+.fa:before,
+.gmf-icon:after,
+.gmf-icon:before {
+  font-size: 2.4rem;
```

Comment:
I feel icons "data", "tools" and "geolocation" are too big, in comparison with font size of menu options. I would suggest: 

`font-size: 2.0rem`

and 

`button {
   width: 2.1em;
   height: 2.1em;
 }`